config/paths.js

```diff
@@ -19,9 +19,7 @@ var isInNodeModules = (
 );
 
 // Are we developing create-react-app locally?
-var isInCreateReactAppSource = (
-  process.argv.some(arg => arg.indexOf('--debug-template') > -1)
-);
+var isInCreateReactAppSource = process.argv.indexOf('--debug-template') !== -1;
```

Comment:
This won't work, actually, there's a reason it was with `some` beforehand!

`process.argv` might be `['something', '--debug-template --smoke-test']`, which `process.argv.indexOf` doesn't return true for. `indexOf` only checks if an _entire string_ matches, not if _part of a string_ matches.